src/app/interfaces/dataset-details.interface.ts

```diff
@@ -0,0 +1,44 @@
+import { IscsiExtentType } from 'app/enums/iscsi.enum';
+import { Dataset } from 'app/interfaces/dataset.interface';
+
+export interface DatasetDetails extends Dataset {
```

Comment:
It seems duplicate https://github.com/truenas/webui/blob/08a980c2b262a35b02296079d64cd71182c62800/src/app/interfaces/dataset.interface.ts#L163-L189